packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -475,10 +475,49 @@ export default class Meetings extends WebexPlugin {
      * @param {String} [options.feedbackId] ID used for tracking
      * @param {String} [options.locusId]
      * @param {Stirng} [options.correlationId]
+     * @param {String} [options.meetingId]
      * @returns {String} feedback ID logs were submitted under
      */
-    uploadLogs(options = {}) {
-      return this.loggerRequest.uploadLogs(options);
+    uploadLogs(options = {autoUploadLogs: true}) {
```

Comment:
Checking the config for auto upload here seems to be out of place. The check should be added before this function is called, not as part of the function.